api-analytics/index.js

```diff
@@ -0,0 +1,93 @@
+var ENDPOINT = 'api/quotations';
+
+// Register the worker and show the list of quotations.
+navigator.serviceWorker.register('worker.js').then(function() {
+  loadQuotations();
+});
+
+// When clicking add button, get the new quote and author and post to
+// the backend.
+document.getElementById('add').onclick = function() {
```

Comment:
Sort of a nit but can we use a real `<form>` and do `onsubmit` so users can add a new quote upon `enter` key?